src/manual/devel/namespaces.md

```diff
@@ -188,6 +188,7 @@ Worker types are specific to the provisioner ID, but provisioners that provide g
 * `tutorial` - default worker type for the getting-started tutorial in this documentation
 * `github-worker` - default worker type for Github-triggered jobs
 * `hg-worker` - default worker type for Mercurial-triggered jobs
+* `buildbot-bridge` - worker type for tasks to be executed using the Buildbot Bridge
```

Comment:
I don't know if this belongs here, since it's not within provisionerId `aws-provisioner-v1`.  What do you think?  Maybe we should combine workerTypes and provisionerIds into a single section and list them hierarchically?

This touches on an area of increasing urgency, which is to have a way of enumerating existing provisionerId/workerType pairs and associating them with a worker implementation.